Valet.podspec

```diff
@@ -6,6 +6,7 @@ Pod::Spec.new do |s|
   s.homepage = 'https://github.com/square/Valet'
   s.authors  = 'Square'
   s.source   = { :git => 'https://github.com/square/Valet.git', :tag => s.version }
+  s.pod_target_xcconfig = { 'SWIFT_VERSION' => '4.0' }
```

Comment:
I wouldnt use that. There is an official DSL `s.swift_version` that you can use.